Networking/Sources/Networking/Models/Products/Product.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  File.swift
+//  
+//
+//  Created by Omar Ahmed on 23/06/2022.
+//
+
+import Foundation
+
+public struct Product: Codable {
+    let id: Int?
+    let title: String?
+    let price: Double?
+    let welcomeDescription: String?
+    let category: String?
+    let image: String?
+    let rating: Rating?
+    
+    enum CodingKeys: String, CodingKey {
+        case id = "id"
+        case title = "title"
+        case price = "price"
+        case welcomeDescription = "welcomeDescription"
+        case category = "category"
+        case image = "image"
+        case rating = "rating"
+    }
```

Comment:
Since properties has the same key names, no need to add `CodingKeys` if you're going to remove `init(from:)`